packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/AxisInfoTextLabels.jsx

```diff
@@ -25,7 +25,7 @@ export default () => {
   return (
     <React.Fragment>
       <div id="tooltip" className="initial-tooltip" />
-      <div id="axisInfoTextLabelsBargraph"> </div>
+      <div id="axisInfoTextLabelsBargraph" className="example-graph-container" />
```

Comment:
You could encapsulate this in a little react component such that in the future we don't have to touch so many components with the same change.

Call it ExampleGraphContainer and have it apply the styles.

```jsx
import React from 'react';
import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';

export default (props) => (
    <div id={props.id} className="example-graph-container" />
  )};
```

Reply:
fixed here: b96918dd5f39518a197e75a4ebf3293dab90785f